src/components/ui/inputs/TextInput.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+import { useId } from 'react';
+import { clsx } from 'clsx';
+
+type TextInputProps = {
+  name?: string;
+  type?: string;
+  className?: string;
+  label: string;
+  id: string;
+  placeholder?: string;
+};
+
+export function TextInput({
```

Comment:
Why do we call it Text Input if it can accept different types? Like can we use it for number input or password input?)) 
If yes, lets just call it Input or CustomInput

Reply:
Sure